src/wazuh_modules/wm_control.c

```diff
@@ -58,28 +58,60 @@ char* getPrimaryIP(){
         }
     }
 
+    OSHash *gateways = OSHash_Create();
```

Comment:
It is necessary to create this table in Linux and call to `getGatewayList`?

Actually, here are the warnings when building the linux agent:
```
wazuh_modules/wm_control.c: In function ‘getPrimaryIP’:
wazuh_modules/wm_control.c:62:9: warning: implicit declaration of function ‘getGatewayList’; did you mean ‘getIfaceslist’? [-Wimplicit-function-declaration]
     if (getGatewayList(gateways) < 0){
         ^~~~~~~~~~~~~~
         getIfaceslist
wazuh_modules/wm_control.c:65:14: warning: unused variable ‘gate’ [-Wunused-variable]
     gateway *gate;
                     ^~~~
```

And finally, the compilation fails:
```
libwazuh.a(wm_control.o): In function `getPrimaryIP':
/root/wazuh/src/wazuh_modules/wm_control.c:62: undefined reference to `getGatewayList'
collect2: error: ld returned 1 exit status
Makefile:1140: recipe for target 'ossec-agentd' failed
make[1]: *** [ossec-agentd] Error 1
make[1]: *** Waiting for unfinished jobs....
libwazuh.a(wm_control.o): In function `getPrimaryIP':
/root/wazuh/src/wazuh_modules/wm_control.c:62: undefined reference to `getGatewayList'
collect2: error: ld returned 1 exit status
Makefile:1118: recipe for target 'ossec-logcollector' failed
make[1]: *** [ossec-logcollector] Error 1
libwazuh.a(wm_control.o): In function `getPrimaryIP':
/root/wazuh/src/wazuh_modules/wm_control.c:62: undefined reference to `getGatewayList'
collect2: error: ld returned 1 exit status
Makefile:1102: recipe for target 'ossec-execd' failed
make[1]: *** [ossec-execd] Error 1
libwazuh.a(wm_control.o): In function `getPrimaryIP':
/root/wazuh/src/wazuh_modules/wm_control.c:62: undefined reference to `getGatewayList'
collect2: error: ld returned 1 exit status
Makefile:1233: recipe for target 'ossec-syscheckd' failed
make[1]: *** [ossec-syscheckd] Error 1
make[1]: Leaving directory '/root/wazuh/src'
Makefile:568: recipe for target 'agent' failed
make: *** [agent] Error 2
```